docs/api/1.x/authentication.rst

```diff
@@ -4,385 +4,121 @@ Authentication
 
 .. _authentication:
 
-A word about users
-==================
+Kinto is an API, and uses the request headers to authenticate the current user.
 
-First of all, Kinto **doesn't provide users management**. There is no such thing
-as user sign-up, password modification, etc.
+Based on the authentication enabled in configuration, Kinto will authenticate the user and assign a :term:`user identifier` to the request (eg. ``ldap:alice@corp``).
 
-However, since Kinto handles permissions on objects, users are uniquely
-identified.
+Since authentication is entirely pluggable in Kinto (see :ref:`configuration <configuration-authentication>`), the HTTP method used to authenticate requests may vary.
+But the main methods you are most likely to encounter will read the ``Authorization`` header. For example:
 
-How is that possible?
----------------------
+* :ref:`Kinto Accounts <api-accounts>` expects a valid username and password as *Basic Auth*
+* :ref:`OpenID Connect <api-openid>` will validate a *Bearer Access Token* against an :term:`Identity Provider`
 
-Kinto uses the request headers to authenticate the current user.
+If the authentication fails, the server will return a |status-401| error response. It can also return a |status-403| error response, which would mean that the operation performed on the resource is not allowed to unauthenticated users :)
```

Comment:
I'm a bit confused because here it seems like failed authentication should be obvious (401/403) but elsewhere the documentation says to request `/v1/` and try to infer that auth has failed because of no principals. Is that specific to `/v1/`? Does it depend on the auth mechanism?

Reply:
You are right I'll rephrase this!